client/components/single-product-plan/index.jsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import SingleProductPlanPriceGroup from './price-group';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class SingleProductPlan extends Component {
+	static propTypes = {
+		billingTimeFrame: PropTypes.string,
+		currencyCode: PropTypes.string,
+		description: PropTypes.oneOfType( [ PropTypes.string, PropTypes.object ] ),
+		discountedPrice: PropTypes.number,
+		fullPrice: PropTypes.number,
+		isPurchased: PropTypes.bool,
+		subtitle: PropTypes.oneOfType( [ PropTypes.string, PropTypes.object ] ),
+		title: PropTypes.oneOfType( [ PropTypes.string, PropTypes.object ] ),
+	};
+
+	renderHeader() {
+		const {
+			billingTimeFrame,
+			currencyCode,
+			discountedPrice,
+			discountedPriceRange,
```

Comment:
It was my intention. However since the [`<PlanPrice>`](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/plan-price/index.jsx#L27-L28) component accepts currently props like `rawPrice` and `rawPriceRange`, I went the same interface.

We could potentially update the `<PlanPrice>` component definition so that the `rawPrice` can be either number or an array of numbers.

Reply:
I see, but since our component is a wrapper, it makes sense. I understand why you want to keep the same interface, but keeping the same interface comes with the tradeoff of 2 more props, which IMHO is worse.

Reply:
I've updated the `<PlanPrice>` interface in the [latest commit](https://github.com/Automattic/wp-calypso/pull/36721/commits/153f5c1249893b4e124203592fbade1e693615f5).